src/compiler-debugging.md

```diff
@@ -150,65 +150,94 @@ Cool, now I have a backtrace for the error!
 ## Getting logging output
 [getting-logging-output]: #getting-logging-output
 
-These crates are used in compiler for logging:
+The compiler uses the [`tracing`] crate for logging.
 
-* [log]
-* [env-logger]
+[`tracing`]: https://docs.rs/tracing
 
-[log]: https://docs.rs/log/0.4.6/log/index.html
-[env-logger]: https://docs.rs/env_logger
-
-The compiler has a lot of `debug!` calls, which print out logging information
+The compiler has a lot of [`debug!`] calls, which print out logging information
 at many points. These are very useful to at least narrow down the location of
 a bug if not to find it entirely, or just to orient yourself as to why the
 compiler is doing a particular thing.
 
-To see the logs, you need to set the `RUSTC_LOG` environment variable to
-your log filter, e.g. to get the logs for a specific module, you can run the
-compiler as `RUSTC_LOG=module::path rustc my-file.rs`. All `debug!` output will
-then appear in standard error.
+[`debug!`]: https://docs.rs/tracing/0.1/tracing/macro.debug.html
+
+To see the logs, you need to set the `RUSTC_LOG` environment variable to your
+log filter. Your log filter can be just `debug` to get all `debug!` output, or
+`path::to::module` to get *all* output (not just `debug!`) from a particular
+module, or `path::to::module=debug` to get only `debug!` output form a
+particular module.
```

Comment:
Is that true? I thought that `path::to::module=debug` meant all logging for debug and higher, including e.g. info.

Reply:
Yes, you're right! I'll fix that.

Reply:
Fixed.